test/tests.py

```diff
@@ -264,16 +265,22 @@ def test_0005_invalidIncorrectUri(self):
         self.assertEqual(response.status_code, HTTPStatus.METHOD_NOT_ALLOWED)
 
     def test_0006_OSB(self):
-        cmd = ['docker', 'ps', '--format="{{.ID}}"', '--filter', 'name=migration']
-        container_id = subprocess.run(cmd, stdout=subprocess.PIPE, text=True).stdout.strip().replace('"', '')
-
-        if container_id:
-            cmd_exec = f"docker exec {container_id} ./runTestBenchmarks.sh"
-            logger.warning(f"Running command: {cmd_exec}")
+        if self.deployment_type == "cloud":
+            cmd_exec = f"./runTestBenchmarks --unique-id {self.index}"
```

Comment:
The path here for runTestBenchmarks will be in a different directory. It's probably easiest to use `/root/runTestBenchmarks.sh`

Reply:
Could we also use the self.unique_id instead of self.index here? It seems like it may be clearer

Reply:
Good idea, I'll update in the next commit.